source/howtos/updating_pelican_sites.rst

```diff
@@ -0,0 +1,601 @@
+Updating CASS Pelican Websites
+==============================
+
+This documentation applies to both the CASS main website, the OSL website and 
+any other sites using the Pelican static site generator.
+
+What is Pelican
+---------------
+
+`Pelican`_ is a static site generator. In short, it allows us to create a great 
+website by writing simple text files and checking them into Git.
+
+.. _Pelican: http://docs.getpelican.com
+
+What is Git
+-----------
+
+Git is a system that tracks changes to files. It allows multiple people to edit 
+the same set of files and merge their changes together. All the files, and a 
+history of all the changes made to them, are stored in such a way that you can 
+retrieve the past state of any file easily.
+
+We store our Pelican site files in Git, and we use Github to store those git 
+repositories. Github provides an easy to use web interface to the majority of 
+git's functionality. We use it to review changes, to merge changes into 
+production, and to track issues.
+
+Git via GitHub
+~~~~~~~~~~~~~~
+
+You can do most git operations entirely from the Github web UI. If installing a 
```

Comment:
Say something like '*Many people use `git` through the commandline but this isn't everybody's cup of tea. Thankfully* you can do most git operations...